mission_control/include/mission_control/behaviors/payload_command.h

```diff
@@ -0,0 +1,75 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2020, QinetiQ, Inc.
```

Comment:
@patapufate meta: should read 2021? Anyhow, we can fix the entire `experimental` branch right before merging into  `master`

Reply:
Fixed.